### PR TITLE
Update policies

### PR DESCRIPTION
**Pull request by abikouo**
- Remove all defined policies and add a new policy allowing the terminator to describe and delete resources from multiple regions

### CODE REVIEW

File: aws/policy/policy.yaml
Comment (gravesm) [inline aws/policy/policy.yaml:1]: I just noticed that we already have https://github.com/ansible/aws-ci-admin/blob/main/aws/smallset-policy.yaml. Maybe we should just move that here.

Comment (abikouo) [reply aws/policy/policy.yaml:1]: updated

File: aws/smallset-policy.yaml
Comment (gravesm) [inline aws/smallset-policy.yaml:1]: I think this file needs to be in `aws/policy/` to get picked up.

Comment (abikouo) [reply aws/smallset-policy.yaml:1]: By default this terminator version is deployed with the small set policy, see https://github.com/ansible/aws-ci-admin/blob/29aff23c145c679d38dc0391b478c6f207e5d2ac/aws/terminator.yml#L37. However we can update that and move the file into `aws/policy`

Comment (gravesm) [reply aws/smallset-policy.yaml:1]: Yeah, let's do that just to be consistent.

Comment (abikouo) [reply aws/smallset-policy.yaml:1]: updated

**Review by gravesm — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**